modules/schema/src/main/java/org/apache/ignite/internal/schema/configuration/SchemaConfigurationConverter.java

```diff
@@ -230,7 +230,7 @@ public static IndexDefinition convert(TableIndexView idxView) {
      * @return TableIn.
      */
     public static PrimaryKeyDefinition convert(PrimaryKeyView primaryKey) {
-        return new PrimaryKeyDefinitionImpl(Set.of(primaryKey.columns()), Set.of(primaryKey.affinityColumns()));
+        return new PrimaryKeyDefinitionImpl(Set.of(primaryKey.columns()), List.of(primaryKey.colocationColumns()));
```

Comment:
Colocation columns is not check for uniquiness here.

Reply:
Fixed. Check moved to constructor of fhe `PrimaryKeyDefinitionImpl`